### PR TITLE
Update A320_Neo_FCU.html

### PR DESCRIPTION
**Pull request by JackZ974**
Fixes #[issue_no]

**Summary of Changes**

Slightly changed the position  & sizes of text and lines to be closer to the original FCU (this is part of previously closed PR#)

**Screenshots (if necessary)**

**Additional context**

### CODE REVIEW

**Comment by UncleClapton** [comment]
Please provide screenshots?

**Comment by AdenFlorian** [comment]
This looks like it's got some visual issues, the managed ALT crcle is touching the alt number, there's a line going thu the V/S:
image


IRL:
image

**Review by JackZ974 — COMMENTED** [review]
Modified the circle LVL/CHG position. Slightly modified the css file also
image
image

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html [hunk 1]
```diff
@@ -104,3 +105,14 @@
...
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/Approach.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/SelectableElement.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/ContextualMenu.js"></script>
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/Runway.js"></script>
+
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/NavSystem.js"></script>
+
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/Shared/BaseAirliners.js"></script>
+
+<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.js"></script>
```
Comment (AdenFlorian) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:118]: did you mean to add this? I don't think this should be here

**Review by AdenFlorian — CHANGES_REQUESTED** [review]

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html (hunk 1 above)
Comment (JackZ974) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:118]: Guess it was there already. I ddin't added it

Comment (JackZ974) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:118]: Yes  I checked these lines were on the original Asobo code. Remove them if you wish

Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:118]: Can you remove them please?

This file already exists on master branch, and these lines are not on master's version of this file.

Comment (JackZ974) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:118]: ok

**Review by JackZ974 — COMMENTED** [review]
removed extra js that wer present on the original ASobo Code

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html [hunk 2]
```diff
@@ -88,19 +89,5 @@
...
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/Approach.js"></script>
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/SelectableElement.js"></script>
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/ContextualMenu.js"></script>
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/LogicElements/Runway.js"></script>
-
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/NavSystems/Shared/NavSystem.js"></script>
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/Shared/BaseAirliners.js"></script>
 
-<script type="text/html" import-script="/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.js"></script>
```
Comment (AdenFlorian) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:106]: These weren't supposed to be removed, I will fix. I recommend reviewing the diff's of your PR by looking through all the changes on the Files changed tab in your PR, and make sure that the only things that have changed were the things you meant to change.

Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:106]: In your last commit, you removed the part that wasn't supposed to be there, but you removed too much, and ended up removing things that should've stayed. https://github.com/flybywiresim/a32nx/pull/523/commits/22ccb571f6e0571e09e40030f8c32c93116eec35

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html [hunk 3]
```diff
@@ -7,58 +7,58 @@
...
                     </div>
 
-                    <div id="Altitude">
-                        <svg width="125%" height="100%">
-                            <text id="ALT" class="Common Active" x="40%" y="25%">ALT</text>
-                            <text id="Value" class="Common Value" x="20%" y="85%">-----</text>
-                            <circle id="Illuminator" r="5.6%" cx="85%" cy="62%" />
+                    <div id="VerticalSpeed">
+                        <svg width="100%" height="100%">
+                            <circle id="Illuminator" r="7%" cx="6%" cy="62%" />
```
Comment (AdenFlorian) [inline A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:48]: this circle is for the altitude, moving it here broke it so it always displays, I will fix.

Comment (AdenFlorian) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html:48]: looks like a copy paste error that happened when you swapped the Altitude withe the Vertical speed svg, which I dont think was necessary

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html (hunk 2 above)
Comment (JackZ974) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html]: Taht's what reviewers are for, I suppose!
Are we good now?

**Comment by AdenFlorian** [comment]
final looks after my tweaks, top is IRL, middle is this PR, bottom is master branch:
image

If you're happy with this @JackZ974 I'll go ahead and merge it!

File: A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html (hunk 3 above)
Comment (JackZ974) [reply A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.html]: OK. Fix it please

**Review by JackZ974 — COMMENTED** [review]
go

**Review by AdenFlorian — APPROVED** [review]

**Pull request merged**